packages/data/src/store/index.js

```diff
@@ -1,3 +1,19 @@
+export const META_SELECTORS = [
+	'getIsResolving',
+	'hasStartedResolution',
+	'hasFinishedResolution',
+	'isResolving',
+	'getCachedResolvers',
+];
+
+const META_ACTIONS = [
+	'startResolution',
+	'finishResolution',
+	'invalidateResolution',
+	'invalidateResolutionForStore',
+	'invalidateResolutionForStoreSelector',
+];
```

Comment:
Nit: We can move these to `constants.ts` and type them as `const` to get the desired typings.

```ts
export const META_SELECTORS = [
	'getIsResolving',
	'hasStartedResolution',
	'hasFinishedResolution',
	'isResolving',
	'getCachedResolvers',
] as const;
```